Automattic/jetpack
# Social: Migrate Social Image Generator settings initial state

## Proposed changes:

* Migrate Social Image Generator settings initial state

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to Social admin page
* Toggle SIG ON/OFF
* Confirm that the request in the Network tab is a success
* Reload the page
* Confirm that the change is persisted
* Confirm the same on Jetpack settings page
* Create a new post
* Open Jetpack/Social sidebar
* Confirm that the settings updated above are reflected in the editor
* Do a sanity check on WPCOM sites editor
